### DISCUSSION TITLE
Public Sponsor

### DISCUSSION THREAD

**Discussion by enzifiri** [enzifiri/GitHub-Achievements#48] [Q&A] [answered]
Sponsoring open source work via [GitHub Sponsors](https://github.com/sponsors)

**Comment by Kral001** [comment, marked as answer]
Hello

**Comment by Foozee26** [comment]
good idea tnx

**Reply by enzifiri** [reply]
Farklı soruya cevap ver bu soru yanıtlandı hocam
